### PR TITLE
Updating tutorial with commands adding user to snap_microk8s group

### PR DESCRIPTION
**Pull request by Gmerold**
# Description

- Updates tutorial with commands for adding user to the `snap_microk8s` group. Bootstrapping controller will fail if it's not done.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have bumped the version of the library

### CODE REVIEW

File: docs/tutorial/01_bootstrap_juju.md [hunk 1]
```diff
@@ -25,6 +25,13 @@ From your terminal, install Juju.
 sudo snap install juju --channel=3.1/stable
 ```
 
+Add your user to the `snap_microk8s` group:
+
+```console
+sudo usermod -a -G snap_microk8s $USER
```
Comment (gruyaume) [inline docs/tutorial/01_bootstrap_juju.md:31]: Those should probably be in the `install MicroK8s` section (just before the `microk8s enable ...`

File: .readthedocs.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# .readthedocs.yaml
```
Comment (gruyaume) [inline .readthedocs.yaml:1]: This file is unrelated to this PR.

**Review by gruyaume — CHANGES_REQUESTED** [review]

File: .readthedocs.yaml (hunk 2 above)
Comment (Gmerold) [reply .readthedocs.yaml]: It is. I just thought I'll use the occasion to add it.
I'll move it to a separate PR.

**Review by saltiyazan — APPROVED** [review]

**Review by gruyaume — APPROVED** [review]
